js/foundation.dropdownMenu.js

```diff
@@ -286,8 +284,8 @@ class DropdownMenu extends Plugin {
     const $body = $(document.body);
     this._removeBodyHandler();
     $body.on('click.zf.dropdownMenu tap.zf.dropdownMenu', (e) => {
-      var $link = this.$element.find(e.target);
-      if ($link.length) return;
+      var isItself = this.$element.andSelf().find(e.target).length;
```

Comment:
`andSelf()` is deprecated and should be replaced with `addBack()`

Apart from that I think the order is wrong. When you want to include the elem itself it should be
```javascript
var isItself = this.$element.find(e.target).addBack().length;
```

Reply:
I used `$(e.target).closest(this.$element).length`. I think that testing the DOM tree upward is way faster than browsing all the nodes downward.

Reply:
Good idea 👍